autopayroll-CordaService/contracts/src/main/kotlin/com/autopayroll/contracts/MoneyStateContract.kt

```diff
@@ -0,0 +1,26 @@
+package com.autopayroll.contracts
+
+import net.corda.core.contracts.CommandData
+import net.corda.core.contracts.Contract
+import net.corda.core.transactions.LedgerTransaction
+
+// ************
+// * Contract *
+// ************
+class MoneyStateContract : Contract {
+    companion object {
+        // Used to identify our contract when building a transaction.
+        const val ID = "com.autopayroll.contracts.MoneyStateContract"
```

Comment:
Is there a reason why you are using a string constant rather than a class reference?

Reply:
I believe it was a subjective choice made by the platform team. But technically i believe they are interchangeable with the class reference.

Reply:
True. Although using a class reference makes refactoring easier.